toc/rfc/rfc-draft-pcap-bosh.md

```diff
@@ -0,0 +1,59 @@
+# Meta
+[meta]: #meta
+- Name: Integrate pcap feature into BOSH
+- Start Date: 2023-07-06
+- Author(s): @domdom82 @maxmoehl
+- Status: Draft
+- RFC Pull Request: https://github.com/cloudfoundry/community/pull/640
+
+
+## Summary
+
+Members of the [App Runtime Platform WG](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/app-runtime-platform.md) are developing a BOSH release called [pcap-release](https://github.com/cloudfoundry/pcap-release) which adds network traffic capturing tools to both BOSH deployments and Cloud Foundry applications.
+The release was created after an initial discussion was started on the [cf-deployment](https://github.com/cloudfoundry/cf-deployment/issues/980) repository.
+
+Since then, the project has evolved into a BOSH use case and a Cloud Foundry use case. This RFC discusses a proposal to get the BOSH use case integrated with BOSH director and BOSH cli.
```

Comment:
```suggestion
Since then, the project has evolved into a BOSH use case and a Cloud Foundry use case. This RFC discusses a proposal to get the BOSH use case integrated with BOSH director, BOSH agent and BOSH cli.
```